sig-api-machinery/annual-report-2020.md

```diff
@@ -0,0 +1,150 @@
+   10. [priority and fairness to GA](https://github.com/kubernetes/enhancements/issues/1040)
+
+2. Year to date KEP work: What's now stable? Beta? Alpha? Road to alpha?
+   1. Stable
+      1. [Selector index](https://github.com/kubernetes/kubernetes/commit/fea3042f1f84129ab1cb6e481bd51343061673b7) - 1.20
+      2. [Permabeta machinery (sig-arch policy)](https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/1635-prevent-permabeta/README.md) - 1.19
+      3. [Client-go context](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1601-client-go-context/README.md) - 1.18
+      4. [Client-go options](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1601-client-go-context/README.md) - 1.18
+      5. [Dry run](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/576-dry-run/README.md) - 1.18
+      6. [Standardize conditions](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1623-standardize-conditions/README.md) - 1.19
+   2. Beta
+      1. [Priority and fairness](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1040-priority-and-fairness/README.md) - 1.20
+      2. [Selector index](https://github.com/kubernetes/kubernetes/pull/92503) - 1.19
+      3. [Self-link removal](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1164-remove-selflink/README.md) - 1.20
+      4. [Warning headers](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1693-warnings/README.md) - 1.19
+      5. [Server-side apply evolution while in beta](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/555-server-side-apply/README.md) - 1.18, 1.19, 1.20
+   3. Alpha
+      1. [Selector index](https://github.com/kubernetes/kubernetes/pull/87939) - 1.18
+      2. [API server identity](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1965-kube-apiserver-identity/README.md) - 1.20
+      3. [Efficient watch resumption](https://github.com/kubernetes/enhancements/issues/1904) - 1.20
+   4. Pre-alpha
+      1. [Manifest-based admission webhook](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1872-manifest-based-admission-webhooks/README.md) 
+
+
+3. What initiatives are you working on that aren't being tracked in KEPs?
+
+   We are working on mitigating the impact of removing beta APIs in 1.22.
+
+4. What areas and/or subprojects does the group need the most help with?
+
+   The SIG has 2 Working Groups that are largely independent. 
+   
+   There are several areas where regularly the SIG becomes under pressure, especially closer to code freezes and the
+   vast amount of code owned by API Machinery.
+   
+   The ecosystem of the different Kubernetes Clients that we own grows more or less organically. Client-go and
```

Comment:
do you think we should do more outreach in those communities (python, go, etc) for any reason?

Reply:
> do you think we should do more outreach in those communities (python, go, etc) for any reason?

I had not really considered the question.  Does this doc need to reflect future objectives or merely an honest assessment of the current state?